database/migrations/2024_12_20_144244_create_tasks_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('description');
+            $table->enum('status', ['Pending', 'In Progress', 'Completed'])->default('Pending');
+            $table->foreignId('employee_id')->constrained('employees');
```

Comment:
Take a look at this:
https://lightit.slite.page/p/Keh_2qwnzt2jHO/Como-declarar-las-FK-en-las-Migraciones